src/api-errors.js

```diff
@@ -77,3 +77,44 @@ export const invalidParameters = create(10, (message, errors) => {
     message: message || 'invalid parameters'
   }
 })
+
+export const vmMissingPvDrivers = create(11, () => ({
+  message: 'missing PV drivers'
+}))
+
+export const vmIsTemplate = create(12, () => ({
+  message: 'VM is a default template'
```

Comment:
I'm not sure this error is only thrown when deleting a VM default template.

The accuracy/perimeter of the message should match the perimeter of the error.